### ISSUE TITLE
Matrixdynamic questions rowCount is overriden with default value after data is loaded

### ISSUE THREAD

**Issue by ivissani** [surveyjs/survey-library#2018]
### Are you requesting a feature, reporting a bug or asking a question?
Reporting a bug

### What is the current behavior?
When loading answer matrixdynamic questions get their values truncated to the value of rowCount in the question definition.

I've tracked down the bug to the following behaviour. rowCount property (for some unkown to me reason) is set to the number configured in the question definition AFTER data has been loaded (I'm loading survey from service and setting data by doing `survey.data = data`.

So if I defined a question Q with rowCount equal to 1 and I have 2 rows for question Q in my data, when data is loaded first rowCount is correctly set to 2 but afterwards rowCount is set back to 1 thus truncating Q.value to just one element.

For the time being I fixed this bug  by doing the following:

```javascript
survey.onLoadSurveyFromService = function() {
    // Restore matrixdynamic values
    var questions = survey.getAllQuestions();
    for(var i = 0; i < questions.length; i++) {
        var question = questions[i];
        if (question.getType() === 'matrixdynamic') {
            try {
                // answer holds the answer object
                question.value = answer[question.name];
            } catch (e) {
                console.log('Ooopsy');
            }
        }
    }
    // Original behaviour
    survey.render();
};
```

### What is the expected behavior?
Matrixdynamic rowCount value should be set to (at least) the number of rows in data

### Specify your

* browser: Chrome
* surveyjs platform (angular or react or jquery or knockout or vue): jquery
* surveyjs version: 1.5.13

**Comment by andrewtelnov** [comment]
@ivissani I have created [the example](https://next.plnkr.co/edit/1hfnfN88kMdl7448) but I could not reproduce it.
Could you please fork the example and change it so I can reproduce the wrong behavior?

Thank you,
Andrew

**Comment by ivissani** [comment]
Will take a look later, probably on Monday. In the meantime I think this may be a race condition (don't know if that's even posible) because I'm loading survey from service and then setting `sruvey.data` but I did not find a way to "wait" until survey is loaded (no callback or event that I could find). So I'm doing something like the following:

```javascript
let survey = new Survey.Model({
    surveyId: 'some-survey-id-with-matrixdynamic-question',
});
survey.data = answer_with_more_rows_than_default;
```

So I think (at the moment this is just a longshot) that the problem may be that the request to get the the survey is resolved AFTER `survey.data` is assigned.

**Comment by andrewtelnov** [comment]
@ivissani Indeed there is no event that you can use to set the data, so it would not override by model that comes later. I have just added [survey.onLoadedSurveyFromService event](https://github.com/surveyjs/survey-library/issues/2019) that you can use. The event will be available in v1.5.14 that we will release on the next week.

Thank you,
Andrew
